wp-content/themes/pub/wporg-learn-2020/archive-lesson-plan.php

```diff
@@ -187,10 +187,12 @@
 
 <?php else : ?>
 	<section>
-		<div class="row align-middle between section-heading section-heading--with-space">
```

Comment:
I think we could change this layout to use a `col` class to limit the width of the title, something like:

```
		<div class="row gutters between section-heading section-heading--with-space">
			<?php the_archive_title( '<h1 class="section-heading_title h2 col-8">', '</h1>' ); ?>
			<?php get_template_part( 'template-parts/component', 'archive-search' ); ?>
		</div>
```

This is the approach taken with the section intro pattern:

![Screenshot 2023-08-30 at 2 37 54 PM](https://github.com/WordPress/Learn/assets/1017872/3c3563bc-be35-43fb-9252-fb9e0d9a4b86)

Because this is a shared pattern there is more complexity to the change though unfortunately. This style rule is being applied to override the `align-middle` class, when it should actually just be removed. In my tests that seems to be safe. I think we should try to remove this class from the template headings throughout the site, and then remove this style rule too.

```
@media only screen and (min-width: 768px) {
    .section-heading.row {
        align-items:baseline;
    }
}
```